src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -60,6 +60,9 @@ internal abstract class SourceEventSymbol : EventSymbol, IAttributeTargetSymbol
             this.CheckAccessibility(_location, diagnostics, isExplicitInterfaceImplementation);
         }
 
+        public Location Location
```

Comment:
Should we just use `sealed override` on `TryGetFirstLocation()` below and use that instead?

Reply:
can do in followup.